bin/variables/load-aat-environment-variables.sh

```diff
@@ -9,6 +9,7 @@ echo "export SERVICE_AUTH_PROVIDER_API_BASE_URL=http://rpe-service-auth-provider
 echo "export IDAM_API_BASE_URL=https://idam-api.aat.platform.hmcts.net"
 echo "export CCD_IDAM_REDIRECT_URL=https://ccd-case-management-web-aat.service.core-compute-aat.internal/oauth2redirect"
 echo "export CCD_DEFINITION_STORE_API_BASE_URL=http://ccd-definition-store-api-aat.service.core-compute-aat.internal"
+echo "export CAMUNDA_BASE_URL=http://camunda-api-aat.service.core-compute-aat.internal"
```

Comment:
we don't upload to aat env currently in jenkinsfile so not sure this is needed. Upload bpmn file script is only used in 'buildinfra:demo'